hack/components/bump_kubemacpool.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+set -xe
+
+source hack/components/common_functions.sh
+
+KUBEMACPOOL_URL=$(cat components.yaml | shyaml get-value components.kubemacpool.url)
+KUBEMACPOOL_COMMIT=$(cat components.yaml | shyaml get-value components.kubemacpool.commit)
+KUBEMACPOOL_REPO=$(echo ${KUBEMACPOOL_URL} | sed 's#https://\(.*\)#\1#')
+KUBEMACPOOL_PATH=${GOPATH}/src/${KUBEMACPOOL_REPO}
+
+echo 'Fetch kubemacpool sources'
+fetch_component ${KUBEMACPOOL_PATH} ${KUBEMACPOOL_URL} ${KUBEMACPOOL_COMMIT}
+
+echo 'Configure kustomize for CNAO templates and save the rendered manifest under CNAO data'
+(
+    cd ${KUBEMACPOOL_PATH}
+    mkdir -p config/cnao
+
+    cat <<EOF > config/cnao/kustomization.yaml
```

Comment:
Not related to this change, but just wandering why the files are build inline and not static?

Reply:
I think that using heredocs here removes the need to have a kustomization file that is not directly related to the repo. It keeps thing neat and tidy.